Source/SourceGenerators/Blazorise.Generator.Features/README.md

```diff
@@ -0,0 +1,12 @@
+﻿## Code Removal with Fody
+
+Attributes like `GenerateIgnoreEqualityAttribute` are used to signal the source generator to produce additional code. These attributes have no functional purpose in the final compiled code, so they are removed post-compilation using Fody.
+
+If you add a new attribute here, make sure to account for its removal in `Blazorise.Weavers.Fody`.
+
+> It might be worthwhile to place "ready-to-be-removed" attributes in a dedicated namespace, making it easier to remove the entire namespace rather than individual attributes.
+
+## ApiDocsDtos
+
+- Is used by generated code (`ComponentApiDocsGenerator.cs`) and by `Blazorise.Docs` to present the Blazorise API.
```

Comment:
If I understand correctly, if **Fody** works correctly no one can access these objects by using the namespace. i.e `Blazorise.Generator.Features.ApiDocsDtos.ApiDocsForComponent` as these will not exist in the final product?

Reply:
yep.